### ISSUE TITLE
arm64

### ISSUE THREAD

**Issue by vinaooo** [Genymobile/scrcpy#1662]
I'm using scrcpy 1.12.1, it's on ubuntu arm64 repositorys
in snap it is armhf (32 bits) only.

I think snap could be simplest to add an updated arm64.

**Comment by AzureRaptor** [comment]
@rom1v  I would also love to see this.  There are an increasing number of arm64 machines these days - the newer macs IIRC, the Raspberry Pi 3, and definitely the RPi4 (which I am typing this on as we speak :grin: ) all run arm64.  I was very sad when I typed:

`sudo snap install scrcpy`

and received:

`error: snap "scrcpy" is not available on stable for this architecture (arm64)
       but exists on other architectures (amd64, i386).`

Also:
Issue discussed [here](https://github.com/sisco311/scrcpy-snap/issues/23) on scrcpy snap repo.

**Comment by rom1v** [comment]
You could just use a non-snap version, it works on arm;

**Comment by AzureRaptor** [comment]
Non-snap versions are pretty far behind, though, and you don't maintain a
PPA as far as I know.  So snaps are really the best way to acquire scrcpy
right now as far as I can see.

That's an unfortunate side effect of your very hard work in producing so
many bugfixes and feature releases!  :grin:

On Thu, Jul 8, 2021 at 3:20 PM Romain Vimont - ***@***.***
***@***.***>
wrote:

**Comment by rom1v** [comment]
> So snaps are really the best way to acquire scrcpy right now as far as I can see.

https://github.com/Genymobile/scrcpy/blob/master/BUILD.md#simple

**Comment by AzureRaptor** [comment]
> https://github.com/Genymobile/scrcpy/blob/master/BUILD.md#simple

Thank you very much.  Yes, I knew about this, but I think most people are like me, and would prefer to not have to _build_ software to maintain their systems.  There are a lot of things that can break, and if something breaks it can get pretty hairy.  It's just all a little 1999.  :grin: 

But, hey!  Looks like @sisco311 took care of it in the snap repository!

https://github.com/sisco311/scrcpy-snap/issues/23#issuecomment-877830816

So all is good.  This issue can probably be closed now.
